<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>320</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_ac">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plucormin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sq">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>âˆšx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_del">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ympoz">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_zap">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rafens">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 rgb(25, 174, 255), stop:1 rgb(25, 174, 255));



														





														



</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>279</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#evilButton {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>71</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>color:white;


border:2px solid white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgb(236, 236, 236), stop:1 rgb(236,236, 336));



														





</string>
    </property>
    <item>
     <property name="text">
      <string>sin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ctg</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_plucormin</tabstop>
  <tabstop>pushButton_ac</tabstop>
  <tabstop>pushButton_sq</tabstop>
  <tabstop>pushButton_del</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_9</tabstop>
  <tabstop>pushButton_ympoz</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_minus</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_1</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_plus</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_0</tabstop>
  <tabstop>pushButton_zap</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
